actionpack/lib/system_testing/driver_adapters.rb

```diff
@@ -0,0 +1,46 @@
+module SystemTesting
+  # == System Testing Driver Adapters
+  #
+  # By default Rails supports Capybara with the Selenium Driver. Rails provides
+  # configuration setup for using the selenium driver with Capybara.
+  # Additionally Rails can be used as a layer between Capybara and it's other
+  # supported drivers: +:rack_test+, +:selenium+, +:webkit+, or +:poltergeist+.
+  #
+  # *{RackTest}[https://github.com/jnicklas/capybara#racktest]
+  # *{Selenium}[http://seleniumhq.org/docs/01_introducing_selenium.html#selenium-2-aka-selenium-webdriver]
+  # *{Webkit}[https://github.com/thoughtbot/capybara-webkit]
+  # *{Poltergeist}[https://github.com/teampoltergeist/poltergeist]
+  #
+  # === Driver Features
+  #
+  # |                 | Default Browser       | Supports Screenshots? |
+  # |-----------------|-----------------------|-----------------------|
+  # | Rails' Selenium | Chrome                | Yes                   |
+  # | Rack Test       | No JS Support         | No                    |
+  # | Selenium        | Firefox               | Yes                   |
+  # | Webkit          | Headless w/ QtWebKit  | Yes                   |
+  # | Poltergeist     | Headless w/ PhantomJS | Yes                   |
+  module DriverAdapters
```

Comment:
Does it make sense to provide a "base" class to inherit from, so people know which methods they need to implement when creating adapters? 😬